__tests__/app.test.js

```diff
@@ -82,3 +83,27 @@ describe('GET /api/articles/:article_id', () => {
   });
 })
 
+describe('GET /api/articles', () => { 
+  test('200: responds with array of article objects', () => { 
+     
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(( articleData ) => {
+     
+      const body = articleData.body;
+      expect(body).toHaveLength(5);
```

Comment:
Can you explain why you're only expecting 5 articles, and you can explain why it's passing 😭

Reply:
I think this might be your SQL missing articles with no comments, fix this and submit another PR for review!